Fatal error: exception Failure("lvalue compute(5) expected in compute(5)[3] = 5")
